src/NuGetGallery/Services/IContentObjectService.cs

```diff
@@ -12,6 +14,8 @@ public interface IContentObjectService
         ICertificatesConfiguration CertificatesConfiguration { get; }
         ISymbolsConfiguration SymbolsConfiguration { get; }
         ITyposquattingConfiguration TyposquattingConfiguration { get; }
+        Dictionary<string, NuGetPackageGitHubInformation> NuGetPackagesGitHubDependencies { get; }
```

Comment:
I would suggest following the same pattern as `ISymbolsConfiguration`. Create an interface with a method that looks up the GitHub usage for that package.